net.sourceforge.roccat.roccat-tools.yml

```diff
@@ -0,0 +1,68 @@
+app-id: net.sourceforge.roccat.roccat-tools
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+
```

Comment:
there is no `command`.

Reply:
this is the reason of the build failure.

Reply:
The default command is now a bash script which will display the README.md as further setup is required. Maybe we should use a tool like `glow` to format the README on the terminal ?

There is no default `roccat-tools` command as the software is collection of binaries for various devices.

Reply:
this will not work.

Reply:
What do you mean ? How could we improve the situation ?

Based on my tests the default command displays the README in the terminal, and when opening the default desktop file, it opens a new terminal.

Of course it's less than ideal to display these instructions right there, but what are our options here ? It seems other apps needs further setup, like https://github.com/flathub/org.openrgb.OpenRGB as mentioned earlier.